scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,191 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
+    def test_logpmf(self):
+        # Ground truth value from R dmvhyper
+        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
+        assert_allclose(vals1, -1.119814, rtol=1e-6)
+
+        vals2 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=0)
+        assert_allclose(vals2, np.NAN, rtol=1e-8)
+
+        vals3 = multivariate_hypergeom.logpmf(x=[-3, 4], m=[5, 10], n=7)
+        assert_allclose(vals3, np.NINF, rtol=1e-8)
+
+        vals4 = multivariate_hypergeom.logpmf(x=[3, 4], m=[-5, 10], n=7)
+        assert_allclose(vals4, np.NAN, rtol=1e-8)
+
+        vals5 = multivariate_hypergeom.logpmf(x=[[1, 2], [3, 4]],
+                                              m=[[-4, -6], [-5, -10]],
+                                              n=[3, 7])
+        assert_allclose(vals5, [np.nan, np.nan], rtol=1e-8)
+
+        vals6 = multivariate_hypergeom.logpmf(x=[-3, 4], m=[-5, 10], n=1)
+        assert_allclose(vals6, np.nan, rtol=1e-8)
+
+        vals7 = multivariate_hypergeom.logpmf(x=[1, 11], m=[10, 1], n=12)
+        assert_allclose(vals7, np.nan, rtol=1e-8)
+
+        vals8 = multivariate_hypergeom.logpmf(x=[1, 11], m=[10, -1], n=12)
+        assert_allclose(vals8, np.nan, rtol=1e-8)
+
```

Comment:
Would be good to have one above test where x.sum() != n is the _only_ thing that is wrong.
```suggestion

        vals9 = multivariate_hypergeom.logpmf(x=[3, 3], m=[5, 10], n=7)
        assert_equal(vals9, np.NINF)
        
```

Reply:
added this test too.